config/tailwind.config.js

```diff
@@ -204,6 +204,11 @@ module.exports = {
       fontSize: {
         "2xs": ".625rem",
       },
+      keyframes: {
+        'stroke-fill': {
+          to: { 'stroke-dashoffset': 0 },
+        },
+      },
```

Comment:
Credit to @JoshAntBrown since I stole this from #449. I'm not crazy about having to add something to the tailwind config, so if there is a better way to do this, let me know.